src/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  children: React.ReactNode;
+};
+
+export const Button: React.FC<Props> = (props) => {
+  const {
+    children,
+    className,
+    ...rest
+  } = props;
+
+  return (
+    <button
+      type="button"
+      className={classNames('button', className)}
+      {...rest}
+    >
+      {children}
+    </button>
+  );
+};
```

Comment:
it's cool that you decided to make separate components, but I think it only complicated your code in this case

Reply:
This is a good practice before real work, so I get used to writing the way I'm supposed to))